render.c

```diff
@@ -169,7 +169,7 @@ int render(struct mako_state *state, struct pool_buffer *buffer, int scale) {
 
 	if (wl_list_length(&state->notifications) > config->max_visible) {
 
-		height += config->hidden_margin;
+		height += 3;
```

Comment:
Hmm, why?

Reply:
No reason, reverted it back to inner_margin for now.